src/VisualStudio/Core/Def/Implementation/TableDataSource/VisualStudioDiagnosticListTable.BuildTableDataSource.cs

```diff
@@ -229,6 +229,9 @@ public override bool TryGetValue(int index, string columnName, out object conten
                         case ProjectGuids:
                             content = item.ProjectGuids;
                             return ((Guid[])content).Length > 0;
+                        case SuppressionStateColumnDefinition.ColumnName:
```

Comment:
The intention here was for build diagnostics to support suppression. Based on your other comment, I agree we should just not show any value for suppression state for build diagnostics and retain the current behavior. Let me revert this change.